keps/sig-node/1287-in-place-update-pod-resources/README.md

```diff
@@ -99,48 +101,89 @@ Other identified non-goals are:
 ### API Changes
 
 PodSpec becomes mutable with regards to Container resources requests and
-limits. PodSpec is extended with information of resources allocated on the
-Node for the Pod. PodStatus is extended to show the actual resources applied
+limits. PodStatus is extended to show the resources allocated for and applied
 to the Pod and its Containers.
 
 Thanks to the above:
 * Pod.Spec.Containers[i].Resources becomes purely a declaration, denoting the
   **desired** state of Pod resources,
-* Pod.Spec.Containers[i].ResourcesAllocated (new object, type v1.ResourceList)
-  denotes the Node resources **allocated** to the Pod and its Containers,
-* Pod.Status.ContainerStatuses[i].Resources (new object, type
+* Pod.Status.ContainerStatuses[i].ResourcesAllocated (new field, type
+  v1.ResourceList) denotes the Node resources **allocated** to the Pod and its
```

Comment:
FYI a `ResourceList` type is already in use in [Kustomize's configuration functions](https://github.com/kubernetes-sigs/kustomize/blob/master/cmd/config/docs/api-conventions/functions-spec.md#input-type). The GV isn't v1 of course, but I think there's still some potential for confusion.